documentation/api/query/tutorial.markdown

```diff
@@ -323,26 +319,25 @@ certname? It turns out we can, with this fact query:
                                     ["=", "type", "Class"],
                                     ["=", "title", "Apache"]]]]
 
-This may appear a little daunting, so we'll look at it piecewise.
+This may appear a little daunting, so we'll look at it piece by piece.
 
 Let's start with "select_resources". This operator takes one argument, which is
-a resource query, and returns the results of that query, in exactly the form
+a resource query, and returns the results of that query in exactly the form
 you would expect to see them if you did a plain resource query.
 
 We then use an operator called "extract" to turn our list of resources into
-just a list of certnames. So we now conceptually have something like
+just a list of certnames. So we now conceptually have something like:
 
     ["in", "certname", ["foo.example.com", "bar.example.com", "baz.example.com"]]
 
 The "in" operator matches facts whose "certname" is in the supplied list. (For
 now, that list has to be generated from a subquery, and can't be supplied
 directly in the query, so if you want a literal list, you'll unfortunately
-still have to use a combination of "or" and "="). At this point, our query
-seems a lot like the one above, except we didn't have to specify exactly which
+still have to use a combination of "or" and "=".) At this point, our query
```

Comment:
This goes a bit beyond your patch, but this parenthetical isn't actually true anymore. The syntax for this literal list is

```
["in", "certname", ["array", ["foo.example.com", "bar.example.com", "baz.example.com"]]]
```
What would you think of changing "(For now...)" to something in the spirit of

```
(In our case, this list is generated by a subquery. To use an literal list, you must use the the "array" syntax described in the [operators documentation](operators#array).)
```
?